dice_rolling_simulator.py

```diff
@@ -0,0 +1,96 @@
+#Made on May 27th, 2017
+#Made by SlimxShadyx
+
+#Dice Rolling Simulator
+
+import random
+
+#These variables are used for user input and while loop checking.
+correct_word = False
+dice_checker = False
+dicer = False
+roller_loop = False
+
+#Checking the user input to start the program.
+while correct_word == False:
+
+    user_input_raw = raw_input("\r\nWelcome to the Dice Rolling Simulator! We currently support 6, 8, and 12 sided die! Type [start] to begin!\r\n?>")
+
+    #Converting the user input to lower case.
+    user_input = (user_input_raw.lower())
+
+    if user_input == 'start':
+        correct_word = True
+    
+    else:
+        print "Please type [start] to begin!\r\n"
+
+#Main program loop. Exiting this, exits the program.
+while roller_loop == False:
+
+    #Second While loop to ask the user for the certain die they want.
+    while dice_checker == False:
+        user_dice_chooser = raw_input("\r\nGreat! Begin by choosing a die! [6] [8] [10]\r\n?>")
+
+        user_dice_chooser = int(user_dice_chooser)
+
+        if user_dice_chooser == 6:
+            dice_checker = True
+
+        elif user_dice_chooser == 8:
+            dice_checker = True
+
+        elif user_dice_chooser == 12:
+            dice_checker = True
+
+        else:
+            print "\r\nPlease choose one of the applicable options!\r\n"
+    
+    #Another inner while loop. This one does the actual rolling, as well as allowing the user to re-roll without restarting the program.
+    while dicer == False:
+
+        if user_dice_chooser == 6:
+            dice_6 = random.randint(1,6)
```

Comment:
It seems there are too many duplicated codes, could you abstract them (line 68~ 79, line 53~64, etc) to a function.